cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/KafkaCruiseControlServletUtils.java

```diff
@@ -290,6 +292,20 @@ static Pattern topic(HttpServletRequest request) {
     return topicString != null ? Pattern.compile(topicString) : null;
   }
 
+  static Double minValidPartitionRatio(HttpServletRequest request) {
+    String minValidPartitionRatioString = request.getParameter(MIN_VALID_PARTITION_RATIO_PARAM);
+    if (minValidPartitionRatioString == null) {
+      return null;
+    } else {
+      Double minValidPartitionRatio = Double.parseDouble(minValidPartitionRatioString);
+      if (minValidPartitionRatio > 1.0 || minValidPartitionRatio < 0.0) {
+        throw new IllegalArgumentException("The requested minimal partition ratio must be in range [0.0, 1.0] (Requested: "
```

Comment:
1. `minimal` --> `minimum` (the meaning is different).
2. `[0.0, 1.0]` --> `(0.0, 1.0)` -- as `0.0` or `1.0` are not allowed.